test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -281,22 +281,49 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			step( 'Can see the Earn blocks', async function () {
 				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
 				await gEditorComponent.openBlockInserterAndSearch( 'earn' );
+				const shownItems = await gEditorComponent.getShownBlockInserterItems();
+
 				assert.strictEqual(
-					await gEditorComponent.isBlockCategoryPresent( 'Earn' ),
-					true,
-					'Earn (Jetpack) blocks are not present'
+					[
+						'Donations',
+						'OpenTable',
+						'Payments',
+						'Pay with PayPal',
+						'Premium Content',
+						'Pricing Table',
+					].every( ( v ) => shownItems.includes( v ) ),
```

Comment:
Can we move the assertion inside the loop? I mean for clarity if the test fails, so we get direct info on which item was actually not shown in the inserter. Something like:

```js
[
	'Donations',
	'OpenTable',
	'Payments',
	'Pay with PayPal',
	'Premium Content',
	'Pricing Table',
].every( ( item ) =>
	assert( shownItems.includes( item ), `Block inserter doesn't show the ${ item } block` )
);
```

Reply:
Yep, that's indeed better. Thanks for the suggestion.